src/main/kotlin/org/jetbrains/research/testspark/data/CollectorsData.kt

```diff
@@ -0,0 +1,33 @@
+package org.jetbrains.research.testspark.data
+
+import org.jetbrains.research.testspark.collectors.CoverageStatusShowedCollector
+import org.jetbrains.research.testspark.collectors.FeedbackSentCollector
+import org.jetbrains.research.testspark.collectors.GeneratedTestsCollector
+import org.jetbrains.research.testspark.collectors.IntegratedTestsCollector
+import org.jetbrains.research.testspark.collectors.LikedDislikedCollector
+import org.jetbrains.research.testspark.collectors.TestGenerationFinishedCollector
+import org.jetbrains.research.testspark.collectors.TestGenerationStartedCollector
+
+class CollectorsData {
```

Comment:
Now, the class meant to be the collectors' data contains the collectors themselves (i.e., the data owns its owners). Changing the naming or separating the data from the collectors into 2 objects is advisable.